app/code/Magento/Quote/Model/Quote/Item/ToOrderItem.php

```diff
@@ -63,6 +63,16 @@ public function convert($item, $data = [])
             'to_order_item',
             $item
         );
+        if ($item instanceof \Magento\Quote\Model\Quote\Address\Item) {
+            $orderItemData= array_merge(
```

Comment:
Please add a space before "=" and fix the indents for this code block  (as that will fail static tests)